plugins/MantisCoreFormatting/core/MantisMarkdown.php

```diff
@@ -91,54 +91,20 @@ public static function convert_line( $p_text ) {
 	}
 
 	/**
-	 * Customize the logic on Header elements
+	 * Customize the blockHeader markdown
 	 *
 	 * @param string $line The Markdown syntax to parse
 	 * @access protected
-	 * @return void if markdown starts with # symbol | string html representation generated from markdown.
+	 * @return void if markdown starts with # symbol followed by any character | string html representation generated from markdown.
```

Comment:
[This is not valid PHPDoc](https://phpdoc.org/docs/latest/references/phpdoc/tags/return.html). The return type(s) must be defined at the beginning, i.e. `@return string Description`.

Furthermore, it is not correct to use *void* as return type here; as per [PHP documentation](http://php.net/manual/en/function.return.php), when *return* is used without parameter, NULL is returned.

```
* @return string|null HTML representation generated from markdown, or null if
                      markdown starts with # symbol immediately followed by 
                      any character other than space.
```